ColorPicker/ColorPicker.js

```diff
@@ -215,16 +219,41 @@ const ColorPickerExtended = hoc((config, Wrapped) => {
 			};
 		}
 
+		componentDidMount () {
+			this.node = ReactDOM.findDOMNode(this);
+		}
+
 		componentDidUpdate (prevProps) {
-			if (prevProps.value !== this.props.value) {
-				this.hsl = convert.hex.hsl(this.props.value);
+			const {open, value} = this.props;
+			if (prevProps.value !== value) {
+				this.hsl = convert.hex.hsl(value);
 			}
+
+			if (!prevProps.open && open) {
+				on('click', this.handleClick);
+			} else if (prevProps.open && !open) {
+				off('click', this.handleClick);
+			}
+		}
+
+		componentWillUnmount () {
+			off('click', this.handleClick);
```

Comment:
I believe we only need to call this if we unmount while the popup is open, otherwise there's no handler attached

Reply:
There's no need to gate it, you can remove a handler, even if it doesn't exist, without error.